### ISSUE TITLE
Questie Error: Attempt to Call Field

### ISSUE THREAD

**Issue by Rebel-Kat** [Questie/Questie#4990]
## Bug description

First, I hope I am doing this correctly and thank you for this addon.

I did not see any other mention for this but, I am getting this error after the ToC update to Wrath Classic and the last update to Questie has not resolved this. I am fairly new at reporting these things although I have used addons forever. It seems that I cannot pull up the Questie options nor is Questie showing in the addons option of the WoW menu. 

The minimap icon is there and it does pull up the "My Journey" part, but when I click on the option button nothing happens. I have this on my laptop as well as my main PC. Any help would be greatly appreciated.

## Screenshots

![Screenshot 2023-06-23 105814](https://github.com/Questie/Questie/assets/137528230/27759786-5a1d-4f5b-9ad1-46935a56df4a)

![image](https://github.com/Questie/Questie/assets/137528230/3c484b2b-e507-455c-b062-863f18fbeec8)

![image](https://github.com/Questie/Questie/assets/137528230/f7cfb30d-5cdd-4a5a-9e9e-87f52fb98a1e)

## Questie & WoW version

![image](https://github.com/Questie/Questie/assets/137528230/ba5c9e8d-ab6e-492c-a6a5-50256b7ca46f)
WotlK (Classic)

**Comment by MyTechnoHunter** [comment]
You have an sddon that is using the Ace3 libs wrong

Screenshot your addons, not the settings panel
So far, profession master, and 2 others mentioned in the Discord have caused this error

Try disabling your addons 1 by 1 till questie works

**Comment by Rebel-Kat** [comment]
Thank you for the response, I had worked with the addons, but I guessed I missed the bad one, it happened to be a map overlay for information and the zone, the one I have attached the screenshot for below. Once I disabled this one things started working again. I also looked over the Discord which I probably should have done before reporting. But again, thank you for the response and help

![image](https://github.com/Questie/Questie/assets/137528230/85dcd380-c8a7-494b-a20a-4cae81ca760e)
